server/docker-base/template.podfile

```diff
@@ -0,0 +1,6 @@
+platform :{{PLATFORM}}, '{{PLATFORM_VERSION}}'
```

Comment:
Template file used when generating the main Podfile containing the gathered pod dependencies from extensions